packages/braid-design-system/src/lib/components/private/Modal/Modal.tsx

```diff
@@ -236,15 +240,20 @@ export const Modal = ({
           inset={0}
           zIndex="modal"
           pointerEvents="none"
-          transition="fast"
-          opacity={state !== OPEN ? 0 : undefined}
+          transition={
+            // Disable exit transition for center modals
+            position === 'center' && state !== OPEN && state !== OPENING
+              ? undefined
+              : 'fast'
+          }
```

Comment:
Let's pair on this one. Notice that we are still transitioning the backdrop on center exits, might be an opportunity to improve some things.